### DISCUSSION TITLE
merging PR fails with "Merge attempt failed" message, for no apperant reason. My detailed commit message is deleted

### DISCUSSION THREAD

**Discussion by hagai-lvi** [community/community#43181] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

![image](https://user-images.githubusercontent.com/5054459/210322399-bc90ae40-6dbe-4db4-95f9-330fc4de6109.png)
I write detailed commit messages before I'm merging a PR.
sometimes, with no clear explanation, the merge attempt fails.
This is fine, it can happen.
The bad thing about it, is that my  commit message is deleted, and I have no way to restore it in the UI, so that I can copy-paste it and try again.

I would really appreciate it if I can at least see my commit message gryed-out, so that I can copy it to my clipboard, and try merging again

**Comment by LuisContrerasGlz** [comment]
You can use the git log command to view the commit history for your repository, including the commit message for each commit. This will allow you to see the commit message for the commit that was created as part of the merge attempt, even if the merge failed. Alternatively, you can try using a version control system other than Git to manage your commits. Some version control systems, such as Mercurial, store commit messages in a separate file that is not deleted when the commit is deleted. This may allow you to recover your commit message even if the merge fails.

**Reply by hagai-lvi** [reply]
Thanks for you reply @LuisContrerasGlz, **but this is not the situation**.
As I said above, it hapeens `before I'm merging a PR.`
So In my PR I have commit A, commit B and commit C, and I'm about to squash and merge my PR from github's UI.
I'm writing a long commit message, and when I click the merge button on the PR, I'm getting a failure.
**The commit wasn't made**, so I can't find it in the git log

**Comment by mehmet-soyturk-qonic** [comment]
Just got the same issue, and I have had it before multiple times. I have lost my commit message.

**Comment by tashrifbillah** [comment]
Having the same issue with this repository all of a sudden!
https://github.com/AMP-SCZ/utility/pull/106

Edit:
This is interesting. I manually merged it in the backend. After merging, I pushed up the commit and that merged it in the front-end too!
